com.jonathankang.Weibird.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id" : "com.jonathankang.Weibird",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
```

Comment:
Flathub targets stable releases – thus also requires using stable runtimes and project releases.

Reply:
So it's not allowed ot distribute Apps which are build from master branch, right?

Reply:
We allow beta/RC releases of project themselves, but yes, we require stable runtime.

I see you tagged some releases already: https://github.com/JonathanKang/weibird/releases
Do they fail to build against 3.30 runtime?

Reply:
No. I've fixed it using those tag release. Thanks.